src/core/client/stream/tabs/Comments/Stream/PostCommentForm/CreateCommentMutation.ts

```diff
@@ -36,6 +37,7 @@ export type CreateCommentInput = Omit<
 > & {
   commentsOrderBy?: COMMENT_SORT;
   tag?: GQLTAG;
+  connectionKey?: string;
```

Comment:
I'd remove both of these. `tag` is never used except by the code you added in this PR. We can infer both from current state.